clcache.py

```diff
@@ -1334,7 +1334,8 @@ def processDirect(cache, outputFile, compiler, cmdLine, sourceFile):
             origCmdLine = cmdLine
             stripIncludes = False
             if '/showIncludes' not in cmdLine:
-                cmdLine = ['/showIncludes'] + cmdLine
+                cmdLine = ['/showIncludes']
+                cmdLine.extend(origCmdLine)
                 stripIncludes = True
```

Comment:
It complains that

> The list `cmdLine` is extended using reassignment. 

which is not optimal, because we copy everything from `cmdLine` into a new list. While the general advise might make sense, it should not be an issue in our case.

The suggested workarounds are both for extending at the end of the list. But we need to add somthing at the beginning, which we can do by `cmdLine.insert(0, '/showIncludes')`.

Since I don't think it is very readable, I came up with `cmdLine = ['/showIncludes'] + origCmdLine`, which is combining A and B to C, not extending something.